session-1-jigsaw-intro/07_patch_module_option/compile.sh

```diff
@@ -31,5 +32,18 @@ javac --patch-module java.base=src \
       src/java.base/java/util/concurrent/ConcurrentHashMap.java
 
 echo ""
+echo "${info} *** Compiling Main class*** ${normal}"
```

Comment:
Actually, it is used - I removed the main method from the ConcurrentHashMap itself and moved it into the main class. That allows running the code with or without the patch.